src/app/seamlyme/dialogs/me_shortcuts_dialog.ui

```diff
@@ -346,23 +346,27 @@
         </property>
         <property name="html">
          <string notr="true">&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
-&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;title&gt;Seamly2S Shortcuts&lt;/title&gt;&lt;style type=&quot;text/css&quot;&gt;
+&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;title&gt;Seamly2S Shortcuts&lt;/title&gt;&lt;style type=&quot;text/css&quot;&gt;
 p, li { white-space: pre-wrap; }
-&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
+hr { height: 1px; border-width: 0; }
+li.unchecked::marker { content: &quot;\2610&quot;; }
+li.checked::marker { content: &quot;\2612&quot;; }
+&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
```

Comment:
A Segoe UI sneaked in here, which triggers the usual error popup on mac:

![Bildschirmfoto 2023-05-14 um 12 52 33](https://github.com/FashionFreedom/Seamly2D/assets/1392875/5e996ba6-3c52-4c1e-8804-3f785d484d89)

Reply:
@csett86  TY...  I have no clue why Creator sticks the font family in when simply editing the contents of the QTextBrowser  widget. I know I avoid touching the front property when editing forms now, but for some reason the  QTextBrowser still sticks in the font family.  I wish Creator just had a "No font"  item so the system font is used. Guess now I have to remenber to remove any Seqoe references in the editor after saving  in Creator. 

Anyhow... I removed them.